django/core/exceptions.py

```diff
@@ -53,6 +53,16 @@ class DisallowedRedirect(SuspiciousOperation):
     pass
 
 
+class TooManyFieldsSent(SuspiciousOperation):
+    """Request has too many fields to parse"""
+    pass
+
+
+class RequestBodyTooBig(SuspiciousOperation):
+    """Request is too big to parse"""
```

Comment:
Do we also need to note the proviso?... "The size of the request body (excluding any file uploads) ..."